src/admin-views/commerce/reports/orders/summary.php

```diff
@@ -123,6 +123,16 @@
 							<div class="tec-tickets__admin-orders-report__sales-overview__list__item-amount"><?php echo esc_html( sprintf( '%1$s (%2$s)', $sales_totals['total_sales']['price'], $sales_totals['total_sales']['qty'] ) ); ?></div>
 						</div>
 					</div>
+					<?php
+					/**
+					 * Fires after sales breakdown in the Orders Report admin view.
+					 *
+					 * @since TBD
+					 *
+					 * @param WP_Post $post_id The post ID.
```

Comment:
Is this the post ID of the event/CPT with tickets or the post ID of the order itself? Let's clarify that and it's good to go.

Reply:
@juanfra Thanks for catching this. It should be the current post ID.